source/fundamentals/connection/connect.txt

```diff
@@ -18,19 +26,19 @@ example shows each part of the connection URI:
 .. figure:: /includes/figures/connection-string-parts.png
    :alt: Each part of the connection string
 
-In this example, for the protocol, we use ``mongodb+srv`` which specifies the
-:manual:`DNS Seedlist Connection Format
-</reference/connection-string/#dns-seedlist-connection-format>`. This indicates
-that the hostname following it corresponds to the DNS SRV record of your
-MongoDB instance or deployment. If your instance or deployment does not have a
-DNS SRV record, use ``mongodb`` to specify the :manual:`Standard Connection
-Format </reference/connection-string/#standard-connection-string-format>`.
+In this example, we use ``mongodb`` for the protocol, which specifies
+the :manual:`Standard Connection String Format </reference/connection-string/#std-label-connections-standard-connection-string-format>`.
+
+.. tip::
+
+   You can also use the :manual:`DNS Seed List Connection Format
+   </reference/connection-string/#dns-seed-list-connection-format>` if
+   you want more flexibility of deployment and the ability to change the
+   servers in rotation without reconfiguring clients.
```

Comment:
Suggestion: 
I think originally we needed to present the DNS Seedlist Connection Format first because that's what Atlas defaults to. Has anything changed such that we're choosing the non seedlist format first?

Reply:
I see the bullet point on the ticket mentioning moving this to an admonition. This is an old ticket (Aug 2021) that hasn't been kept up to date.
Could you skip this item for this PR? Thanks.

Reply:
I presented it second because the image of the connection string uri changed and now uses the `mongodb` protocol. But, I will move the info back out of the admontion!